### PR TITLE
Add OVM support to belt [Node v10]

### PR DESCRIPTION
**Pull request by krebernisak**
As Optimism currently doesn't not support Node v10 these changes were made a little bit back in time when this repo was on Node v10.

### CODE REVIEW

File: evm-contracts/app.config.ovm.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "contractsDir": "src",
+  "artifactsDir": "abi",
+  "contractAbstractionDir": ".",
+  "useDockerisedSolc": false,
+  "compilerSettings": {
+    "optimizer": {
+      "runs": 1000000,
```
Comment (se3000) [inline evm-contracts/app.config.ovm.json:8]: Do we need a high number of runs on Optimism? Is there any benefit to optimizing contract size for the verifiers?

Comment (krebernisak) [reply evm-contracts/app.config.ovm.json:8]: Yes, you are correct! I took that specific number from somewhere, but can't remember where anymore.

We are optimizing for contract size on OVM L2, so we should set the optimizer to only 1 run.

I just tested this today and both `AggregatorProxy.test.ts` & `FluxAggregator.test.ts` will get an "out of gas" error for 1000000 runs, but they seem to work fine when the compiler optimizer is configured to only do 1 run.

Comment (se3000) [reply evm-contracts/app.config.ovm.json:8]: Nice, let's drop it.

**Comment by krebernisak** [comment]
Closing in favor or [Add OVM support to the toolchain (belt & test-helpers) #3930](https://github.com/smartcontractkit/chainlink/pull/3930).

**Pull request closed without merging**